nixos/doc/manual/development/releases.xml

```diff
@@ -188,6 +188,23 @@
       </listitem>
     </itemizedlist>
   </section>
+  <section xml:id="after-final-release-time">
+    <title>After the final release time</title>
+    <itemizedlist spacing="compact">
+      <listitem>
+        <para>
+          Mark the old stable as <literal>deprecated</literal> or <literal>unsupported</literal> by updating
+          the <literal>state</literal> field in the <literal>.version-state.nix</literal> file to
+          the desired state. It's recommended to also add a <literal>description</literal> field in that file
+          to give the user instructions on how to actually upgrade.
```

Comment:
When marking a release as deprecated, it should likely also include a note in the description about when it will go from deprecated to unsupported.